lib/Extension/LanguageServerPhpstan/Tests/Model/PhpstanLinterTest.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Phpactor\Extension\LanguageServerPhpstan\Tests\Model;
+
+use PHPUnit\Framework\TestCase;
+use Phpactor\Extension\LanguageServerPhpstan\Model\Linter\PhpstanLinter;
+use Phpactor\Extension\LanguageServerPhpstan\Model\PhpstanProcess;
+
+class PhpstanLinterTest extends TestCase
+{
+    public function testLinterUsesTmpFileByDefault(): void
+    {
+        $originalFilePath = '/foo';
+        $expectedFileContent = '<file content that will be written to tmp file>';
+
+        $phpstanProcess = $this->createMock(PhpstanProcess::class);
+        $phpstanProcess->expects($this->once())
+            ->method('analyse')
+            // ->with($this->stringContains(sys_get_temp_dir() . '/phpstanls'));
```

Comment:
remove comment

Reply:
done in https://github.com/phpactor/phpactor/pull/2764/commits/4755a7b189954924d6cd5e8e5c1a5e65514f91c7